### PR TITLE
Error Patterns front end

### PR DESCRIPTION
**Pull request by sophiebpeneva**

### CODE REVIEW

File: frontend/src/app/api.service.ts [hunk 1]
```diff
@@ -82,9 +82,24 @@ export class ApiService {
...
    */
   getEventsData(filename, filters): Observable<object> {
+    const params = this.buildFilter(filters, new HttpParams());
+    return this.http.get(environment.apiRest + 'events/' + filename, {
+      params,
+    });
+  }
+
+  /**
+   * @description This method makes an http call and fetches data that contais information about
```
Comment (peterborocz) [inline frontend/src/app/api.service.ts:92]: Typo: contains

File: frontend/src/app/api.service.ts [hunk 2]
```diff
@@ -82,9 +82,24 @@ export class ApiService {
...
+   * the odds of an error larger than a given threshold for every date that the two files have in common.
+   *
+   * @param string filename1 The name of the first file to take the data from
+   * @param string filename2 The name of the second file with the comaprison data
```
Comment (peterborocz) [inline frontend/src/app/api.service.ts:96]: Typo: comparison

**Review by peterborocz — CHANGES_REQUESTED** [review]

File: frontend/src/app/api.service.ts (hunk 2 above)
Comment (sophiebpeneva) [reply frontend/src/app/api.service.ts]: done

File: frontend/src/app/api.service.ts (hunk 1 above)
Comment (sophiebpeneva) [reply frontend/src/app/api.service.ts]: done

**Review by peterborocz — APPROVED** [review]

**Pull request merged**